app/templates/calendar/layout.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Meta -->
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Assistant:wght@200;300;400;500;600;700;800&display=swap"
+        rel="stylesheet">
+
+    <!-- CSS -->
+    <link href="{{ url_for('static', path='/grid_style.css') }}" rel="stylesheet" type="text/css">
+
+    <!-- Scripts -->
+    <script src="{{ url_for('static', path='/js/grid_scripts.js') }}"></script>
+    <script src="https://unpkg.com/ionicons@5.2.3/dist/ionicons.js"></script>
+    <title>Calendar</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav class="text-lightgray background-darkblue">
+            <div class="menu">
+                <div class="fixed-features">
+                    <div>
+                        <a href="/profile">
+                            <ion-icon name="person-circle-outline"></ion-icon>
+                        </a>
+                    </div>
+                    <div>
+                        <ion-icon name="search" alt="Search icon"></ion-icon>
+                    </div>
+                    <div>
+                        <ion-icon name="settings-outline"></ion-icon>
+                    </div>
+                    <div>
+                        <ion-icon name="calendar-outline"></ion-icon>
+                    </div>
+                    <div>
+                        <ion-icon name="cube-outline"></ion-icon>
+                    </div>
```

Comment:
Please enter a link to the about us page here after that <div> 
with the icon of:
```<ion-icon name="information-circle-outline"></ion-icon>```

put it in a link <a> tag - no need to set any type of class for it.

Reply:
added it to the about page!